### PR TITLE
feat(cross-check): add tip

### PR DESCRIPTION
**Pull request by NikitaKorevo**
**🟢 Add `deploy` label if you want to deploy this Pull Request to staging environment**

#### 🧑‍⚖️ Pull Request Naming Convention

- Title should follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- Do not put issue id in title
- Do not put WIP in title. Use [Draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/) functionality
- Consider to add `area:*` label(s)

* [x] I followed naming convention rules

---

#### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Test Case
- [ ] Documentation update
- [ ] Other

#### 💡 Background and solution

| Before | After |
|-|-|
| image | image |

#### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Database migration is added or not needed
- [x] Documentation is updated/provided or not needed
- [x] Changes are tested locally

### CODE REVIEW

File: client/src/modules/CrossCheck/components/SubmittedStatus.tsx [hunk 1]
```diff
@@ -16,7 +16,8 @@ export function SubmittedStatus(props: Props) {
 
   if (!solution) {
     const deadlinePassedMessage = 'Submission deadline has already passed';
-    const message = `You haven't submitted solution. ${deadlinePassed ? deadlinePassedMessage : ''}`;
+    const tipMessage = 'Try to submit your solution as early as possible';
+    const message = `You haven't submitted solution. ${deadlinePassed ? deadlinePassedMessage : tipMessage}`;
```
Comment (valerydluski) [inline client/src/modules/CrossCheck/components/SubmittedStatus.tsx:20]: As far as I remember, we cannot submit after the deadline, so it seems odd to me to display such a text if submission is not possible.

Comment (valerydluski) [reply client/src/modules/CrossCheck/components/SubmittedStatus.tsx:20]: my bad

File: client/src/modules/CrossCheck/components/SubmittedStatus.tsx [hunk 2]
```diff
@@ -16,7 +16,8 @@ export function SubmittedStatus(props: Props) {
 
   if (!solution) {
     const deadlinePassedMessage = 'Submission deadline has already passed';
-    const message = `You haven't submitted solution. ${deadlinePassed ? deadlinePassedMessage : ''}`;
+    const tipMessage = 'Try to submit your solution as early as possible';
```
Comment (valerydluski) [inline client/src/modules/CrossCheck/components/SubmittedStatus.tsx:19]:
```suggestion
    const tipMessage = 'Try to submit your solution as soon as possible';
```

Comment (NikitaKorevo) [reply client/src/modules/CrossCheck/components/SubmittedStatus.tsx]: @valerydluski done

**Review by valerydluski — APPROVED** [review]

**Pull request merged**